pypose/module/liesplinese3.py

```diff
@@ -0,0 +1,53 @@
+import torch
+from torch import nn
+
+class lieSpline(nn.Module):
+
+    def __init__(self):
+        super().__init__()
+        self.B = torch.tensor([[5, 3, -3, 1],
+                               [1, 3, 3, -2],
+                               [0, 0, 0, 1]]) / 6
+
+    def compute_weights(self, time):
+        alpha = torch.arange(4)
```

Comment:
this will have issues if inputs tensors is on GPU.

Reply:
Have modified the code 

`alpha = torch.arange(4, dtype=time.dtype, device=time.device)`